### ISSUE TITLE
Border/Background highlight is not available for tags in tagpicker

### ISSUE THREAD

**Issue by mdmdakbar** [OfficeDev/office-ui-fabric-react#2462]
### Bug Report

- __Package version(s)__:  4.25.1002
- __Browser and OS versions__:  Edge

#### Priorities and help requested (not applicable if asking question):

Requested priority: (Blocking, High, Normal, Low)   Normal

Products/sites affected: VSTS 

#### Describe the issue:

On high contrast, tags present are not showing any border or highlighting.

#### Actual behavior:

Border/Background highlight is not available for tags in tagpicker on using high contrast.

#### Expected behavior:

Border/Background highlight should be visible for each tags in tagpicker on  high contrast.

Attachments: 

![image](https://user-images.githubusercontent.com/19491487/29171717-105f81d4-7dfb-11e7-8f33-f6a1d33f5d74.png)
